php/Terminus/User.php

```diff
@@ -73,6 +75,12 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
+  public function instruments() {
```

Comment:
Encapsulation is paramount. Opening variables up to public use begs for misuse and bad code. Only the class should be able to touch its own variable.